.circleci/config.yml

```diff
@@ -40,6 +49,15 @@ jobs:
           name: rails db:setup
           command: bundle exec bin/rails db:wait db:setup
 
+      - run:
+          name: rails db:setup for docker image
+          command: docker-compose run -e RAILS_ENV=test zync bin/rails db:drop db:create db:migrate
```

Comment:
Avoid using docker compose when not necessary

Just build the image `docker build . --tag=$IMAGE_NAME`
Then use this image by `docker run $IMAGE_NAME`

It is just a smoke test not a full testing